myapp/app/assets/stylesheets/components/_task_details.scss

```diff
@@ -0,0 +1,17 @@
+.task_details__item_row {
```

Comment:
[Don't have to update codes]
Just so you know, I think I rarely see the `.hoge__fuga` class name in rakuma application, we use `.hoge_fuga`.
In this training, it's up to you!